src/controllers/user.controller.js

```diff
@@ -0,0 +1,18 @@
+const userServices = require('../services/user.services');
+const customError = require('../utils/customError');
```

Comment:
Remove import if not used

Reply:
I have fixed the import and adding to catch error when any field is not given

Reply:
I also added a test with it